srcpkgs/rocksdb/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'rocksdb'
+pkgname=rocksdb
+version=5.17.2
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release -DPORTABLE=1 -DWITH_TESTS=0 -DUSE_RTTI=0 -DWITH_LZ4=1"
+makedepends="liblz4-devel"
+short_desc="RocksDB is a embeddable, persistent key-value store for fast storage"
+maintainer="magenbluten <mb0@codemonkey.cc>"
+license="GPL-2.0-or-later,Apache-2"
```

Comment:
you are including LICENSE.leveldb which is BSD-3-Clause, it should also be in the license file.

Reply:
oh yes. actually i think facebook totally screwed the licenses. i did not include it because the readme says you can choose between gpl or apache-2 at will:

RocksDB is dual-licensed under both the GPLv2 (found in the COPYING file in the root directory) and Apache 2.0 License (found in the LICENSE.Apache file in the root directory). You may select, at your option, one of the above-listed licenses.